tools/flang1/utils/symtab/symini.cpp

```diff
@@ -1541,6 +1541,7 @@ const char *SyminiFE90::init_names0[] = {
     "any",
     "compl",
     "count",
+    "isnan",
```

Comment:
Do you know whether isnan should be added to all these init_names arrays or only one of them is necessary?

Reply:
I'm seeing it added to other init_names too.